### ISSUE TITLE
Unclear if focus indicator counts when determining if the focus is obscured

### ISSUE THREAD

**Issue by rachaelbradley** [w3c/wcag#3950]
2.4.11 and 2.4.12 reference the "user interface component" but the definition of this does not explicitly include or exclude the focus indicator as part of the component. Assuming the intent is that the focus indicator is not part of the component, some of the language in the understanding documents makes this unclear. The understanding documents also do not explicitly state that an external focus indicator should not count towards focus not obscured. 

This causes an issue both in passing 2.4.11 and in failing 2.4.12.

**Comment by detlevhfischer** [comment]
I think it is pretty clear that the the SC normative text, "When a [user interface component](https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum.html#dfn-user-interface-component) receives keyboard focus...",  the UIE is the element that the SC focuses on, not whatever focus indication is used on it. 

Whenever the focus is some outline around the UIE, the focus will then also not be fully obscured. The SC misses cases where the focus indicator may be an underline or run inside the perimeter of the UIE. Taking the normative text narrowly, content would then seem to pass if the UIE - say, a link - is partly visible but not its focus (like the underline). I have read through the 2.4.11 Understanding text and see nothing that contradicts this interpretation, and nothing that suggests the focus indicator should be conasidered part of the UIE. 

@rachaelbradley Which bits of understanding text do you think are unclear?

One could certainly strengthen the understanding by advising that neither the UIE not its focus indcation should be fully obscured, but that is not a "must" - that would need a normative change.

**Comment by patrickhlauke** [comment]
the main point of confusion ... is in the name of the SC itself. "focus not obscured" on its face implies that it means the focus indicator, particularly since the spec uses "focus" to mean "focus indicator" in other SCs like "focus visible" and "focus appearance". too late now, of course, but "focused *element* not obscured" would have avoided any misunderstanding...

**Comment by patrickhlauke** [comment]
The understanding for both Focus Not Obscured (Minimum) and Focus Not Obscured (Enhanced) now contain a note that tries to clarify the situation explicitly:

> Note
>
> This criterion evaluates the focused component, rather than the focus indicator. The component itself does not include the focus indicator when checking that "the component is not entirely hidden" - unless the focus indicator is inside the component, or focus is indicated by a change to the component itself. Although users benefit from both the component and the focus indicator (if external to the component) not being obscured when tracking the focus, for the purposes of this criterion only checking the component provides a clearer metric. However, if the focus indicator is fully obscured, it would likely fail [2.4.7 Focus Visible](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible).

https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum.html
https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-enhanced

This addition was made in https://github.com/w3c/wcag/pull/4104 and https://github.com/w3c/wcag/pull/4403

Closing this issue as resolved. However, note that I also just filed a speculative PR for a potential WCAG 2.3, where the potential confusion should be mitigated by renaming the SCs to "Focussed Component Not Obscured" (as "Focus..." is already used in other SCs to refer to the indicator) https://github.com/w3c/wcag/pull/5183
